plugins/experimental/cache_fill/cache_fill.cc

```diff
@@ -72,7 +72,10 @@ cont_check_cacheable(TSHttpTxn txnp)
     return false;
   }
   int lookupStatus;
-  TSHttpTxnCacheLookupStatusGet(txnp, &lookupStatus);
+  if (TSHttpTxnCacheLookupStatusGet(txnp, &lookupStatus) == TS_ERROR) {
+    TSError("[cache_fill][%s] Couldn't get cache status of object", __FUNCTION__);
```

Comment:
Better to refer to PLUGIN_NAME rather than the literal "cache_fill". See:
https://github.com/apache/trafficserver/pull/10394/files#diff-896f422b7d09fb7725036af2885bbbca071b822167a1f8a99fbcb3ccd9d46f40R124

Also, it's probably better to not include the `__FUNCTION__` in an error message. You already worded the message to be unique enough to identify where the Error is coming from, so we should be in good shape as far as that goes.